static/js/browser/kg.js

```diff
@@ -705,7 +705,14 @@ async function renderKGPage(
   // Add "More" to existing cards.
   util.appendMoreToAll();
 
-  if (NO_POP_TYPES.includes(type)) return;
+  const loadingElem = document.getElementById("page-loading");
```

Comment:
nit - move this into the if block since it's not used otherwise

Reply:
Done